lib/formfields/admin/ipsandports/formfield.ipsandports_add.php

```diff
@@ -16,28 +16,30 @@
  */
 
 return array(
-	'ipsandports_add' => array(
-		'title' => $lng['admin']['ipsandports']['add'],
-		'image' => 'icons/ipsports_add.png',
+	'ipsandports_edit' => array(
```

Comment:
did you mix up ipsandports_add and ipsandports_edit??? There is no `$result` when adding new entities